doc/source/reference/arrays.classes.rst

```diff
@@ -648,12 +669,15 @@ N-dimensional enumeration
 Sometimes it may be useful to get the N-dimensional index while
 iterating. The ndenumerate iterator can achieve this.
 
->>> for i, val in np.ndenumerate(a):
-...     if sum(i)%5 == 0: print(i, val)
-(0, 0, 0) 10
-(1, 1, 3) 25
-(2, 0, 3) 29
-(2, 1, 2) 32
+.. try_examples::
+
+  >>> import numpy as np
+  >>> for i, val in np.ndenumerate(a):
+  ...     if sum(i)%5 == 0: print(i, val)
```

Comment:
While we are at it, is it worth putting the print on it's own line ?

Reply:
Addressed in 7b18a43b39c046ae6d9aefb8296f5d146201d637.